health/micro-ui/web/micro-ui-internals/packages/modules/health-payments/src/components/custom_comp/filter_section.js

```
@@ -0,0 +1,135 @@
+  };
+
+  useEffect(() => {
+    const data = Digit.SessionStorage.get("paymentInbox");
+    if (data?.selectedProject) {
+      setProjectSelected(data?.selectedProject);
+    }
+  }, []);
+
+  useEffect(() => {
+    if (project.length == 0) {
+      let datak =
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'datak' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)